### ISSUE TITLE
Safari is not showing image

### ISSUE THREAD

**Issue by murilobd** [cstefanache/angular2-img-cropper#252]
Hi,
I don't know why, but on Safari image is not being displayed on canvas! How could I debug this to see if I can fix this issue?

**Comment by sch3bal** [comment]
Hey, I am having the same issue.
Are there any Solutions?

**Comment by jcbowyer** [comment]
Yes i am having the same issue on mobile devices.   If I set a regular image src it appears but if I set the cropper image property it is blank.
